sql/moz-fx-data-shared-prod/firefox_ios_derived/retention_v1/query.sql

```diff
@@ -0,0 +1,33 @@
+SELECT
+  metric_date,
+  first_seen_date,
+  app_name,
+  normalized_channel,
+  country,
+  adjust_ad_group,
+  adjust_campaign,
+  adjust_creative,
+  adjust_network,
+  COUNTIF(ping_sent_metric_date) AS ping_sent_metric_date,
+  COUNTIF(ping_sent_week_4) AS ping_sent_week_4,
+  COUNTIF(active_metric_date) AS active_metric_date,
+  COUNTIF(retained_week_4) AS retained_week_4,
+  COUNTIF(retained_week_4_new_client) AS retained_week_4_new_clients,
+  COUNTIF(new_client_metric_date) AS new_clients_metric_date,
+  COUNTIF(repeat_client) AS repeat_clients,
+FROM
+  firefox_ios.retention_clients
+WHERE
+  metric_date = DATE_SUB(@submission_date, INTERVAL 27 DAY)
+  AND submission_date = @submission_date
+  AND NOT is_suspicious_device_client
```

Comment:
this filter is mainly for funnel metrics, therefore we need to leave it as an additional column on this table to allow people to use it or not.